src/components/category-box/category-box-styles.module.scss

```diff
@@ -61,6 +62,7 @@
   line-height: 30px;
```

Comment:
Maybe not related with this PR but there are some updated styles. The closed boxes default state has an arrow, do you think it is possible to add it on this PR?
![image](https://user-images.githubusercontent.com/6906348/59518668-6921c380-8ec6-11e9-9350-d8afcfb699df.png)

Reply:
I can change it in this PR, no problem :+1:

Reply:
Actually, it's not that simple, because these two icons are different. The double-arrow icon has his own padding and different size, so trying to keep this in this aligned with a single arrow icon it's a nightmare. I need a coherent version of both arrow from @fannycc, I'll skip it in this PR.
![Screenshot from 2019-06-14 18 08 02](https://user-images.githubusercontent.com/15097138/59525875-62e41500-8ecf-11e9-840b-4bcb51c0ab31.png)
![Screenshot from 2019-06-14 18 07 35](https://user-images.githubusercontent.com/15097138/59525877-64add880-8ecf-11e9-9b48-831ceb8bfe8e.png)